docs/upgrade/0.3.5-to-0.3.6.rst

```diff
@@ -91,11 +91,16 @@ Greeter*.
      git stash pop
 
 #. Make sure you have Ansible installed. If running ``which ansible`` returns 
-   ``ansible not found``, you should ::
+   nothing, you should ::
```

Comment:
Huh, it says "foobar not found" on my Mac, but it's reasonable to expect differences between BSD `which` (on Mac OS X) and Linux's `which`. Anyway. Thanks for double-checking on both version families of Tails, @ageis 💯